contrib/test_jenkins.sh

```diff
@@ -1058,7 +1058,7 @@ run_gtest() {
 		GTEST_TOTAL_SHARDS=1 \
 		GTEST_FILTER=malloc_hook_cplusplus.mallopt \
 		make -C test/gtest test
-	(cd test/gtest && rename .tap _mallopt_gtest.tap malloc_hook_cplusplus.tap && mv *.tap $GTEST_REPORT_DIR)
+	(cd test/gtest && rename .tap _mallopt_gtest.tap malloc_hook_cplusplus.tap && mv *.tap $GTEST_REPORT_DIR) || true
```

Comment:
why?

Reply:
well, the right solution is to mark the test as skipped in the .tap file.. let's try to do it if it's not too complicated

Reply:
do you mean to create fake(empty) `test_case.test` and call `UCS_TEST_SKIP_R` instead of not adding to GTEST registry?

Reply:
actually, maybe we should skip the test by calling UCS_TEST_SKIP from the inherited constructor, instead of not adding to registry? in here fore example: https://github.com/openucx/ucx/pull/3462/files#diff-81a6b8fd8e30591ae79cfd430ad27fdcR176

Reply:
or: override a (new) virtual function in ucs::test_base such as "bool is_skip_test()", and call this function from test_base::SetUpProxy()

Reply:
> actually, maybe we should skip the test by calling UCS_TEST_SKIP from the inherited constructor, instead of not adding to registry? in here fore example: https://github.com/openucx/ucx/pull/3462/files#diff-81a6b8fd8e30591ae79cfd430ad27fdcR176

should we pass a reason to `UCS_TEST_SKIP_COND_F`?

Reply:
maybe we can stringify the macro argument

Reply:
done

> or: override a (new) virtual function in ucs::test_base such as "bool is_skip_test()", and call this function from test_base::SetUpProxy()

this opens new opportunities to skip that tests that require call `GetParam->...` or etc to understand whether it needs to skip the test or not.
I'll skip more test in the next round of the review

Reply:
> I'll skip more tests in the next round of the review

@dmitrygladkov let's keep this for next PR

Reply:
@yosefe ok, sure
the PR seems to be ready for the review